extensions/blocks/google-calendar/test/utils.js

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
Mainly added these tests to help get my head around how the regex was working, and they did help to highlight a potential circular dependency issue, but not sure if they add enough value to warrant keeping - open to other's thoughts on this.